pkg/appconsts/appconsts.go

```diff
@@ -98,6 +98,11 @@ const (
 	// The min gas price acts as a filter. Transactions below that limit will not pass
 	// a nodes `CheckTx` and thus not be proposed by that node.
 	DefaultMinGasPrice = 0.1
+
+	// MinimumDataCommitmentWindow is a constant that defines the minimum allowable window for the QGB data commitments.
+	// Any data commitment window set to a value lower than this constant will trigger an error.
+	// The value of this constant can be configured in genesis, through governance proposals, or in upgrades.
```

Comment:
On the other hand the `DataCommitmentWindow` is a parameter so I think it may make sense to specify that in the comment on the line above... Something like:

```go
	// MinimumDataCommitmentWindow defines the minimum valid value for the x/qgb parameter DataCommitmentWindow.
```

Then we can remove line 2 above the "...trigger an error."

[non-blocking][question] Why define this in `appconsts.go` instead of `x/qgb`?

Reply:
> [non-blocking][question] Why define this in appconsts.go instead of x/qgb?

That's where I saw all the constants being defined. I can move it to the module if makes more sense

Reply:
`appconsts` is useful for constants that span multiple modules to avoid import cycles but IMO we should avoid it for constants that are contained to one module. In other words, I think its preferable for constants to be defined in the module that they are used.